serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-applications.adoc

```diff
@@ -0,0 +1,132 @@
+= Building and Deploying Production Ready Workflows with the Operator
```

Comment:
Not sure about the term production ready?  Wondering if it gives people the impression things are supported?

Reply:
What term do you suggest? They must mirror. So if we have the "Development Profile", then we have "Production Profile". Mutable/Immutable?

> Building and Deploying Workflows in Production environments with the Operator

?

Reply:
For now, I've removed any "production" mentions and left only "build and deploy".